File: 1542.py
while True:
    try:
        Q,D,P = input().split(' ')
        Q=int(Q)
        D=int(D)
        P=int(P)
        pages = (Q * D * P) / (P - Q)
        if pages<=1:
            print("%d pagina"%pages)
        else:
            print("%d paginas"%pages)
    except:
        break

File: 1542.c
#include <stdio.h>

int main() {
 
    int q, d, p, paginas;

    while(scanf("%d", &q) != 0){
        if(q==0){
            break;
        }
        scanf("%d %d", &d, &p);
        paginas= (q*d*p)/(p-q);

        if(paginas > 1){
            printf("%d paginas\n", paginas);
        }
        else{
            printf("%d pagina\n", paginas);
        }
    }
 
    return 0;
}